src/DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -462,14 +462,14 @@ private function normalizeDriverOptions(array $connection): array
      *
      *  doctrine_mongodb:
      *     mappings:
-     *         MyBundle1: ~
-     *         MyBundle2: xml
-     *         MyBundle3: { type: attribute }
-     *         MyBundle4: { type: xml, dir: Resources/config/doctrine/mapping }
-     *         MyBundle5:
+     *         App1: ~
```

Comment:
This configs need to be updated, as `dir` is required when outside of a bundle: https://github.com/symfony/symfony/blob/7bbc9be7ea7d607fa901edb2068c2cea13d24978/src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php#L68

Reply:
I was wondering if we should keep these examples here if we have them in the documentation

Reply:
Having an example for attribute and xml, and for a bundle would be enough.